src/test/java/pl/antonina/tasks/child/ChildCreator.java

```diff
@@ -0,0 +1,35 @@
+package pl.antonina.tasks.child;
+
+import pl.antonina.tasks.user.Gender;
+import pl.antonina.tasks.user.UserData;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+class ChildCreator {
+    static ChildData createChildData() {
+        ChildData childData = new ChildData();
+        childData.setGender(Gender.FEMALE);
+        childData.setName("Name");
+        childData.setBirthDate(LocalDate.of(2017, 02, 02));
+        UserData userData = new UserData();
+        final String randomEmail = UUID.randomUUID().toString() + "@gmail.com";
```

Comment:
Probably not needed here, but this is interesting project: https://github.com/DiUS/java-faker

Reply:
ok, I added a new issue, after this PR I will check it.